### PR TITLE
[CWS] add cws-intrumentation binary to the package

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

Add cws-instrumentation to the agent package. It can be used on non ebpf platform.

### CODE REVIEW

**Review by spikat — APPROVED** [review]

**Review by paulcacheux — APPROVED** [review]

**Review by chouquette — APPROVED** [review]

File: omnibus/config/software/datadog-agent.rb [hunk 1]
```diff
@@ -183,6 +183,13 @@
     move 'bin/agent/dist/security-agent.yaml', "#{conf_dir}/security-agent.yaml.example"
   end
 
+  # CWS Instrumentation
+  cws_inst_support = (not heroku_target?) && (linux_target?)
```
Comment (alopezz) [inline omnibus/config/software/datadog-agent.rb:187]: Nit
```suggestion
  cws_inst_support = !heroku_target? && linux_target?
```

**Pull request merged**